tree/ntuple/v7/inc/ROOT/RMiniFile.hxx

```diff
@@ -224,8 +228,9 @@ public:
    std::uint64_t WriteNTupleFooter(const void *data, size_t nbytes, size_t lenFooter);
    /// Writes a new record as an RBlob key into the file
    std::uint64_t WriteBlob(const void *data, size_t nbytes, size_t len);
-   /// Reserves a new record as an RBlob key in the file.
-   std::uint64_t ReserveBlob(size_t nbytes, size_t len);
+   /// Reserves a new record as an RBlob key in the file. If keyBuffer is specified, it must be written *before* the
+   /// returned offset. (Note that the array type is purely documentation, the argument is actually just a pointer.)
+   std::uint64_t ReserveBlob(size_t nbytes, size_t len, unsigned char keyBuffer[kBlobKeyLen] = nullptr);
```

Comment:
This is an interface extension that is needed to implement parallel writing of the `RBlob` keys. I'm starting to experiment with this for distributed writing and I believe it's what we need, but I can also keep it out of this PR for now, just let me know.

Reply:
I think it's fine. Anyway internal API.